plugin/emacs/flycheck-quicklintjs.el

```diff
@@ -0,0 +1,101 @@
+;;; flycheck-quicklintjs --- Quick Lint JS flycheck support   -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;; This flycheck exetension configures flycheck to run Quick Lint JS
```

Comment:
Is it "Flycheck" or "flycheck"

Reply:
In their website, documentation and github they refer with a upper case F, so i tried to maintain that, i missed this one.
ps: but not in lisp code, because gnu lisp style never uses upper cases letters, at least i never saw one, since this is a comment it should be upper case indeed.